src/ui/theme/spacing.ts

```diff
@@ -52,4 +52,9 @@ export const DEFAULT_SPACING: Spacing = {
     tablet: "3rem",
     desktop: "3rem",
   },
+  gapStripeElement: {
```

Comment:
@alfondotnet I did this to match a bit more closer the spacing between the `LinkAuthenticationStripeElement` and the `PaymentElement` to the input elements within the `PaymentElement` from Stripe. Not super elegant but I guess it's fine?